testing-modules/junit-5/src/test/java/com/baeldung/MultipleExtensionsUnitTest.java

```diff
@@ -0,0 +1,28 @@
+package com.baeldung;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import com.baeldung.extensions.EmployeeDatabaseSetupExtension;
+
+public class MultipleExtensionsUnitTest {
+
+    @Order(1)
+    @RegisterExtension
+    static EmployeeDatabaseSetupExtension SECOND_DB =
+            new EmployeeDatabaseSetupExtension("jdbc:h2:mem:DbTwo;DB_CLOSE_DELAY=-1", "org.h2.Driver", "sa", "");
```

Comment:
On continuations, just indent two spaces (not eight). This applies to this file and the next.